### ISSUE TITLE
How would i render a window using vulkan?

### ISSUE THREAD

**Issue by Axilot** [raysan5/raylib#1390]
Hello there,
Im completely new to everything and id like to know if i am able to use vulkan for my renderings?

**Comment by raysan5** [comment]
@Axilot Sorry, raylib does not support Vulkan.

**Comment by bosoni** [comment]
I have checked Think-Silicon's GLOVE (GL->Vulkan) and ANGLE.

GLOVE was quite slow when I tested its benchmarks (and seems to be sleeping state(or dead) so ANGLE is better way to go (it would be great having gl, d3d11, vulkan, metal platforms, just using gles2 codepath).

UWP uses ANGLE, then raylib runs on D3D11. So if using same method and compile ANGLE to use Vulkan, and using these compiled .libs when compiling own game and using 2 .dll:s (libGLES2 and some other, just like in UWP).

One more platform (some(or lots of)  #ifdefs must be added) but I think it is possible, and checking how UWP is made is a good start. Seems that GLFW can open Vulkan window so no need to use SDL. 

@raysan5 What you think, is it worth it (asking because dont know how much #ifdefs needed)?
